sklearn/tree/tests/test_export.py

```diff
@@ -248,6 +267,8 @@ def test_graphviz_toy():
         "}"
     )
 
+    assert contents1 == contents2
```

Comment:
This filling missing assertion is not in a scope of this PR. So if this is not appropriate in the  point of view of reviewing process, I will remove it from this PR and make new PR for this fix.